travis/samples/.jhipster/TestEntity.json

```diff
@@ -1,40 +1,40 @@
 {
     "relationships": [
         {
-            "relationshipName": "manyToOneEntity",
-            "otherEntityName": "relationshipTestManyToOneEntity",
+            "relationshipName": "testManyToOne",
+            "otherEntityName": "testManyToOne",
```

Comment:
I think it would be good to test the case when relationshipName and otherEntityName differ. They are easily confused in the code and then the bug won't spear if they are the same.